src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/config/JacksonConfiguration.java

```diff
@@ -15,6 +16,7 @@
     public ObjectMapper objectMapper(Jackson2ObjectMapperBuilder builder) {
         ObjectMapper objectMapper = builder.createXmlMapper(false).build();
         objectMapper.registerModule(new Jdk8Module());
+        objectMapper.registerModule(new JavaTimeModule());
```

Comment:
Are we sure, because it differs a little bit? We use PropertyNamingStrategy.SNAKE_CASE in the link you have provided, but the message format is PropertyNamingStrategy.UPPER_CAMEL_CASE like defined in here: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1460561625#WorkAllocationPhase2ScopeofDelivery-NewMessageQueueCandidatesDatabaseStructure

Reply:
Yes it is a different structure from the case event message.  Maybe I confused with the JacksonConfiguration used for the API?  Is that not using this one?